crates/red_knot_python_semantic/src/types.rs

```diff
@@ -1778,6 +1778,68 @@ impl<'db> Type<'db> {
                         let revealed_ty = binding.first_parameter().unwrap_or(Type::Unknown);
                         CallOutcome::revealed(binding, revealed_ty)
                     }
+                    Some(KnownFunction::StaticAssert) => {
+                        let parameter_ty = binding.first_parameter().unwrap_or(Type::Unknown);
+                        let truthiness = parameter_ty.bool(db);
+
+                        binding.set_return_ty(Type::none(db));
```

Comment:
Is this line necessary? Shouldn't the stub return type suffice here?

Reply:
No, thanks.